kb/additional-resources.md

```diff
@@ -1,16 +1,23 @@
 ---
-id: external-resources
-title: External Resources
-sidebar_label: External Resources
+slug: additional-resources
+title: Additional resources
+tags:
+  - resources
+date: 2022-12-12T00:00:00Z
 ---
 
-There are many ways to introduce and teach Temporal based on what background you are coming from. Temporal doesn't have a monopoly on explaining Temporal.
+<!-- truncate -->
 
-Here, we list great blogposts and videos by Temporal staff and users that may be helpful in increasing your understanding or convincing your team.
+There are many ways to introduce and teach Temporal based on your background.
+Temporal doesn't have a monopoly on explaining Temporal.
 
-If you have made a tutorial or explanation of Temporal, please [submit them here](https://github.com/temporalio/documentation/edit/master/docs/external-resources.md) so that others can find it!
+Here, we list great blogposts and videos by Temporal staff and users that might be helpful in increasing your understanding or convincing your team.
 
-## Press Announcements
+If you have made a tutorial or explanation of Temporal, please [submit them here](https://github.com/temporalio/documentation/edit/master/kb/additional-resources.md) so that others can find it!
```

Comment:
```suggestion
If you have made a tutorial or explanation of Temporal, please [submit them here](https://github.com/temporalio/documentation/edit/master/kb/how-to-explain-temporal.md) so that others can find it!
```